sources/contract.spec.ts

```diff
@@ -529,7 +529,129 @@ describe("JettonMinter", () => {
 
         expect(await deployerJettonWallet.getJettonBalance()).toEqual(initialJettonBalance);
     });
+    describe('Bounces', () => {
+        // This is borrowed from stablecoin, and is not implemented here.
+        // Should it be implemented?
```

Comment:
why is it a comment here and not an open issue?

Reply:
Should I remove it?

Reply:
it's better to understand what it means and try to deal with it